### PR TITLE
[4.0] custom admin articles menu

### PR DESCRIPTION
**Pull request by brianteeman**
removes the useless reference to descriptions and the associated strings

### testing
create a custom admin menu
creats an admin menu of type articles list

### before
image

### after
image

### CODE REVIEW

**Comment by wilsonge** [comment]
I think access needs to change as the label if we do this to viewing access level. Because otherwise without the description it implies a access group rather than a View Access

**Comment by infograf768** [comment]
Or just leave the description for that one. Easier.

**Comment by brianteeman** [comment]
On my phone so can't check but what do other admin menu do?

**Comment by brianteeman** [comment]
On my phone so can't check but what do other admin menu do?

File: administrator/language/en-GB/com_menus.ini [hunk 1]
```diff
@@ -9,19 +9,12 @@ COM_MENUS_ACTION_DESELECT="Deselect"
 COM_MENUS_ACTION_EXPAND="Expand"
 COM_MENUS_ACTION_SELECT="Select"
 COM_MENUS_ADD_MENU_MODULE="Add a module for this menu"
-COM_MENUS_ADMIN_ACCESS_DESC="Filter by viewing access level."
 COM_MENUS_ADMIN_ACCESS_LABEL="Access"
```
Comment (Quy) [inline administrator/language/en-GB/com_menus.ini:12]: Change to `Viewing Access Level`?

Comment (brianteeman) [reply administrator/language/en-GB/com_menus.ini:12]: No because if you look the column is called access

**Comment by brianteeman** [comment]
> I think access needs to change as the label if we do this to viewing access level. Because otherwise without the description it implies a access group rather than a View Access

The problem is that the column it is referring to is called Access. 
Managing the items displayed here is done in the menu item "Access"

So if you want to change the name from access then it should be done everywhere.

This PR is correct for the current interface of joomla

**Comment by infograf768** [comment]
Let's leave the description. Why systematically and blindly take off descriptions under the pretext of a "current interface" ?
In some cases, the description is just redundant, but here it IS useful...

**Comment by brianteeman** [comment]
lets stay in the dark age as well

**Comment by infograf768** [comment]
Referencing to the "Dark ages" is just a kind of Godwin's law for some people who have no more argument to oppose to simple common sense statements.

**Comment by brianteeman** [comment]
Look at the options you get when you use this. It is absolutely clear that this is viewing levels. Just as it is absolutely clear that the access filter for articles is for viewing levels.

image

image

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4593cffb4c802f84a5443c6ec17ac2f948c245a1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28362">issues.joomla.org/tracker/joomla-cms/28362</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 4593cffb4c802f84a5443c6ec17ac2f948c245a1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28362">issues.joomla.org/tracker/joomla-cms/28362</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28362">issues.joomla.org/tracker/joomla-cms/28362</a>.</sub>

**Pull request merged**